adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -825,6 +823,10 @@
                 configUrl = 'keycloak.json';
             } else if (typeof config === 'string') {
                 configUrl = config;
+            } else if (typeof config === "object") {
```

Comment:
Be carefull here as `typeof null === "object"` as well.

Reply:
Good to know, but isn't this already handled by `if (!config) {` a few lines above?

Reply:
Yeah, should be!